README.md

```diff
@@ -33,65 +32,198 @@ Only time will tell if making our product roadmap
 
 > _**Note**: we don't expect a **single** app to magically "**solve**"
 all of these problems, and we are **not** going
 to "tackle" all the problems at **once**;
 we may end up only having time/resources to solve **one**
 of these challenges.
 This list is intended as a guide not a "gospel"._
+-->
 
 Our immediate plan is to build a **_single_ app**
 that satisfies _several_ human needs
-by focussing on "personal effectiveness".
-We will build simple UI/UX that helps people
-achieve _all_ of their personal goals: Checklists.
-GOTO: [`checklists.md`](https://github.com/dwyl/product-roadmap/blob/master/checklists.md)
+while focussing on the theme of "***personal effectiveness***".
+
+The @dwyl App will be a "_hybrid_"
+containing several features
+that together help the individual
+achieve _all_ of their personal goals
+through a systematic workflow.
+
+The @dwyl App Will have the following basic workflow:
+***Capture, Categorise Complete***.
+
+1. ***Capture*** - capture everything that is on your mind in a "brain dump".
+2. ***Categorise*** - split out blocks of text
+  into distinct items and _categorise_ them. <br />
+  **a)** What type of info/item is it?
+  e.g: Todo Item, Reading list item, shopping list,
+  random thought, bucket list? <br />
+  **b)** Do you have an _existing_ category/tag for the item
+  or is it **`new`**? <br />
+  **c)** Contextualise the item in terms of time/space:
+  a shopping list item context could be "grocery store"
+  whereas a reading list item could be "daily commute"<br />
+  **d)** **Add** any additional ***detail*** you can think of at the time.
+  e.g: acceptance criteria, deadline, dependencies <br />
+  **e)** If the item is a task that has sub-tasks, create and link them.
+3. ***Complete*** - do the work to _complete_ the item/task.<br />
+  **a)** Re-read the original description
+  and determine if you have everything you need to start the task.
+  If not, write down what is missing. <br />
+  **b)** ***Delegate*** if necessary/possible <br />
+  **c)** ***Track progress*** of the task. <br />
+  **d)** ***Confirm*** it was completed to expected standard. <br />
+  **e)** ***Review*** & ***Reflect*** - confirm the task was completed
+  and all the original criteria were met by the assignee.
+  Leave a brief feedback comment with any thoughts.
+  Check the item off as "Done" if appropriate.
+
+For the first step (***Capture***),
+we will build an ultra-simple interface
+to allow people to
+input **`palintext`**
+and save it to a backend as **`text`**.
+For details, please see:
+[`brain-dump.md`](https://github.com/dwyl/product-roadmap/blob/master/checklists.md)
+
+Once the text input is working,
+we will iterate and improve it
+in several interesting and innovative ways
+in response to feedback from people _using_ the MVP. <br />
+We will add additional methods
+to help people "capture" information.
+e.g:
++ Linking to external sources of information.
```

Comment:
```suggestion
+ Linking to external sources of information
```
Removing the dot to be consistent with the other points. Or should we add them to the other lines?